01_todo_app/web/src/components/landing/index.jsx

```diff
@@ -1,68 +1,133 @@
-import React, {useEffect} from 'react';
-import {useDispatch, useSelector} from "react-redux";
-import './index.css';
-import {addMessage, getMessages, updateMessage, deleteMessage} from '../../store/index';
+import React, { useEffect, useState } from "react"
+import { useDispatch, useSelector } from "react-redux"
+import "./index.css"
+import { addMessage, getMessages, updateMessage, deleteMessage } from "../../store/index"
+import Footer from "./Footer"
 
 const Index = () => {
-  const messages = useSelector(state => state.app.messages)
-  const dispatch = useDispatch()
+    const [task, setTask] = useState("")
+    const messages = useSelector((state) => state.app.messages)
+    const dispatch = useDispatch()
 
-  const addNewTodo = async (event) => {
-    event.preventDefault();
-    const data = new FormData(event.target);
-    dispatch(addMessage(data));
-  }
+    const addNewTodo = async (event) => {
+        event.preventDefault()
+        const data = new FormData(event.target)
+        dispatch(addMessage(data))
+    }
 
-  const updateTodo = async (event, msg) => {
-    event.preventDefault();
-    dispatch(updateMessage(msg.id));
-  }
+    const updateTodo = async (event, msg) => {
+        event.preventDefault()
+        dispatch(updateMessage(msg.id))
+    }
 
-  const deleteTodo = async (event, msg) => {
-    event.preventDefault();
-    dispatch(deleteMessage(msg.id));
-  }
+    const deleteTodo = async (event, msg) => {
+        event.preventDefault()
+        dispatch(deleteMessage(msg.id))
+    }
 
-  useEffect(() => {
-    dispatch(getMessages())
-  }, []);
+    useEffect(() => {
```

Comment:
For the sake of consistency, please include prettier here and deal with the formatting :)

Reply:
Prettier on my vscode for some reason differs from what you expect from installing the package. On my end it was formatted, but I have installed the prettier package and run the format command.